FirebaseDynamicLinks/Sources/Public/FirebaseDynamicLinks/FIRDynamicLinks.h

```diff
@@ -33,6 +33,8 @@ NS_ASSUME_NONNULL_BEGIN
  * @class FIRDynamicLinks
  * @abstract A class that checks for pending Dynamic Links and parses URLs.
  */
+
+NS_EXTENSION_UNAVAILABLE_IOS("Firebase Dynamic Links is not supported for iOS extensions.")
```

Comment:
@eldhosembabu or @chliangGoogle  can you think of any situations where this would break existing iOS users?

Reply:
This has always failed on extension builds. See #6548